src/components/Modal/Modal.js

```diff
@@ -29,30 +29,47 @@ const Modal = ({ showModal, handleModalClose, deleteItem, item }) => {
         }
       }
     };
+
+    const closeOnClickOutside = (e) => {
+      if (e.target.classList.contains('dialog')) {
```

Comment:
I see you changed the className from `dialog-backdrop` to `dialog`. I think it might be helpful if we keep it `dialog-backdrop` to help keep this function clear that when you click on the backdrop/overlay with the 0.3 opacity, it closes the modal. It took me a second to understand what element actually closes the modal. I do see that .dialog has its own CSS as well, so maybe there is still a reason to keep the .dialog class on that overlay div

Reply:
@connietran-dev I'm following BEM conventions to be consistent, so the block is dialog representing the whole set of things, and everything inside is an element of dialog. I don't want to abandon our convention after coming this far, so any suggestions for naming that would still fit? Some ideas:

- `dialog-with-backdrop` and `dialog-with-backdrop__element`
- `dialog-w-backdrop` and `dialog-w-backdrop__element`
- `dialog-backdrop` and `dialog-backdrop__element`
- `dialog-overlay` and `dialog-overlay__element`
- leave it as `dialog` and put a comment in the CSS above it

Reply:
Ah, makes sense. We should definitely use .dialog as the whole set. I like dialog-background!